disnake/asset.py

```diff
@@ -34,12 +34,17 @@
 from . import utils
 from .errors import DiscordException, InvalidArgument
 
-__all__ = ("Asset",)
+__all__ = (
+    "AssetMixin",
```

Comment:
this shouldn't be here

Reply:
cc3f3fc22552f4f439545ea6d7997b8a3725709d